experiments/ClimaEarth/user_io/arg_parsing.jl

```diff
@@ -59,6 +59,7 @@ function get_coupler_args(config_dict::Dict)
     FT = config_dict["FLOAT_TYPE"] == "Float64" ? Float64 : Float32
 
     # Vector of TOML files containing model parameters
+    config_dict["coupler_toml"] = joinpath.(pkgdir(ClimaCoupler), config_dict["coupler_toml"])
```

Comment:
This prevents people from using TOMLs outside of the install pkgdir, which could be in the `.julia` folder, a hidden folder that users should not be expected to touch.

Reply:
This was already the behavior for the TOML files passed to ClimaAtmos, this PR just applies the same behavior further up to ensure the paths are the same for the bucket. 
https://github.com/CliMA/ClimaCoupler.jl/blob/main/experiments/ClimaEarth/components/atmosphere/climaatmos.jl#L451

Reply:
Could we fix this in atmos too? One way would be to introduce a hierarchy: First we try resolving `config_dict["coupler_toml"]` as a path, if such path doesn't exist, and we try prepending `pkgdir(ClimaCoupler)` and resolving that path.